.github/workflows/test-package.yml

```diff
@@ -52,13 +52,6 @@ jobs:
         run: dart test/setup.dart
       - name: Run VM tests
         run: dart test --platform vm
-      - name: Collect coverage
-        run: ./tool/coverage.sh
-      - name: Upload coverage
-        uses: coverallsapp/github-action@v1.1.2
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          path-to-lcov: lcov.info
```

Comment:
Are we planning to manually upload coverage?

Reply:
When I copied this rule to make the `mac-test` rule below, I accidentally copied this collect coverage stuff too, but we should only have one of these rules collecting coverage. So in this PR I had to delete one of them, and I decided to delete the linux coverage collector, because the mac one also collects coverage for the ObjC tests.